opendcs-rest-api-jetty/src/main/java/org/opendcs/odcsapi/jetty/Start.java

```diff
@@ -107,7 +105,8 @@ public static void main(String[] args)
 		appLogger.info("================ Starting ===============");
 
 		// Initialize the JETTY server and servlet holders.
-		Server server = new Server();
+		org.eclipse.jetty.server.Server server = new org.eclipse.jetty.server.Server();
```

Comment:
1. Why?
2. just delete comment code, don't leave it in.

Reply:
Adam and I were just going through the process of pushing these changes in git via IntelliJ.

I am cleaning it up and will be pushing the cleaned code soon.

Reply:
ah, okay.